proposals/compatibility-mode.md

```diff
@@ -107,15 +107,18 @@ During `createRenderPipeline()` and `createRenderPipelineAsync()`, `GPUDepthSten
 The differences in limits between compatibility mode and standard WebGPU
 are as follows
 
-
 | limit                               | compat  | standard  | gl limit                                     |
 | :---------------------------------- | ------: | --------: | :------------------------------------------- |
 | `maxColorAttachments`               |       4 |         8 | min(MAX_COLOR_ATTACHMENTS, MAX_DRAW_BUFFERS) |
 | `maxComputeInvocationsPerWorkgroup` |     128 |       256 | MAX_COMPUTE_WORK_GROUP_INVOCATIONS           |
 | `maxComputeWorkgroupSizeX`          |     128 |       256 | MAX_COMPUTE_WORK_GROUP_SIZE                  |
 | `maxComputeWorkgroupSizeY`          |     128 |       256 | MAX_COMPUTE_WORK_GROUP_SIZE                  |
 | `maxInterStageShaderVariables`      |      15 |        16 | MAX_VARYING_VECTORS                          |
-| `maxStorageBuffersPerShaderStage`   |       4 |         8 | min(GL_MAX_SHADER_STORAGE_BUFFER_BINDINGS, GL_MAX_VERTEX_SHADER_STORAGE_BLOCKS, GL_MAX_FRAGMENT_SHADER_STORAGE_BLOCKS, GL_MAX_COMPUTE_SHADER_STORAGE_BLOCKS) |
+| `maxStorageBuffersPerShaderStage`   |       4 |         8 | min(MAX_SHADER_STORAGE_BUFFER_BINDINGS, MAX_COMPUTE_SHADER_STORAGE_BLOCKS) |
+| `maxStorageBuffersInVertexStage`    |       0 | (ignored) | min(MAX_SHADER_STORAGE_BUFFER_BINDINGS, MAX_VERTEX_SHADER_STORAGE_BLOCKS) (see restriction 18) |
+| `maxStorageBuffersInFragmentStage`  |       0 | (ignored) | min(MAX_SHADER_STORAGE_BUFFER_BINDINGS, MAX_FRAGMENT_SHADER_STORAGE_BLOCKS) (see restriction 19) |
+| `maxStorageTexturesInVertexStage`   |       0 | (ignored) | TODO (see restriction 18) |
+| `maxStorageTexturesInFragmentStage` |       0 | (ignored) | TODO (see restriction 19) |
```

Comment:
do you know what these 2 should be? @greggman @SenorBlanco

Reply:
`MAX_VERTEX_SHADER_STORAGE_BLOCKS` and `MAX_VERTEX_IMAGE_UNIFORMS` respectively